package.json

```diff
@@ -36,6 +36,7 @@
     "bootstrap": "^4.3.1",
     "classnames": "^2.2.6",
     "jquery": "^3.5.1",
-    "popper.js": "^1.15.0"
+    "popper.js": "^1.15.0",
+    "postcss-css-variables-extract": "^0.1.2"
```

Comment:
Probably I should move this to the dev dependencies if we only need it during build